doc/source/templates/configs/compute/cpu/aws_release_test.yaml

```diff
@@ -1,3 +1,6 @@
+cloud_id: {{ env["ANYSCALE_CLOUD_ID"] }}
```

Comment:
Hmm would it make sense if our release test infra automatically populated this field if it's not set? Thinking about how we can reduce the need for files in the future...

Reply:
Yeah, I had to duplicate it just for this reason. Would appreciate that!

Reply:
Ah I took another look at other `yaml` files and it seems like we use this same syntax to populate arbitrary values other than `cloud_id`  e.g. `post_build_cmds`. So it might be non-trivial to solve this generically. Let's follow up on this later.